src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts

```diff
@@ -168,23 +168,24 @@ At this point, you now have the Dockstore CLI set up for interacting with the Do
 
 #### Part 6 - Install cwltool (Optional)
 Dockstore relies on [cwltool](https://github.com/common-workflow-language/cwltool) - a reference implementation of CWL - for local execution of tools and workflows described with CWL.
-You'll need to have Python 3 and [pip3](https://pip.pypa.io/en/latest/installing/) to be installed on your machine.
+You'll need to have Python 3 and [pipx](https://pipx.pypa.io/latest/installation/) to be installed on your machine.
 
 **Note:** cwltool must be available on your PATH for the Dockstore CLI to find it.
 
 You can install the version of cwltool that we've tested for use with Dockstore using the following commands:
 1. Install cwltool
 \`\`\`
 curl -o requirements.txt "${this.dsServerURI}/metadata/runner_dependencies?client_version=${this.dockstoreVersion}&python_version=3"
-pip3 install -r requirements.txt
+pipx install cwltool==${this.cwltoolVersion}
+pipx inject cwltool -r requirements.txt --force
```

Comment:
With ubuntu 22.04 and 24.04, I got 
```
# pipx inject cwltool -r requirements.txt --force
usage: pipx [-h] [--version] {install,inject,upgrade,upgrade-all,uninstall,uninstall-all,reinstall,reinstall-all,list,run,runpip,ensurepath,completions} ...
pipx: error: unrecognized arguments: -r

```

command history is 
```
    1  apt update
    2  apt install curl wget pipx
    3  wget https://raw.githubusercontent.com/dockstore/dockstore/bb4d443e1191f0c330d08ca07bd3240adc0f0f91/dockstore-webservice/src/main/resources/requirements/1.15.0/requirements3.txt
    4  pipx install cwltool==3.1.20230201224320
    5  pipx inject cwltool -r requirements.txt --force
    6  head requirements3.txt 
```

Reply:
`pipx --version`?

Reply:
Note the `-r` command from here so unfortunately "works for me" https://pipx.pypa.io/latest/docs/#pipx-inject